### ISSUE TITLE
sql: new columns in crdb_internal.zones table

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#5636]
PR: https://github.com/cockroachdb/cockroach/pull/40493

From release notes:
> The [`SHOW PARTITIONS`](../v19.2/show-partitions.html) statement now displays inherited as well as explicitly set zone configurations for partitions. Also, it is now possible to retrieve full zone configurations via the `full_config_sql` column in the `crdb_internal.zones` table, and the new `zone_id` and `subzone_id` columns in the `crdb_internal.partitions` table point to partitions' corresponding zone configurations in `crdb_internal.zones`. [#40493][#40493] {% comment %}doc{% endcomment %}

**Comment by ericharmeling** [comment]
I changed the name of this issue to cover what hasn't already been filed/documented (the `crdb_internal` table).
See #5364 and #5541 - these covered the inherited constraints issue.

Several duplicates of this issue have also popped up (#5530, #5531, #5532).

**Comment by rmloveland** [comment]
@ericharmeling FYI I'm addressing the `crdb_internal.zones` part of this issue as part of #4730.

**Comment by rmloveland** [comment]
@ericharmeling I've addressed the following bits re: `crdb_internal.zones` in #5679 

> Also, it is now possible to retrieve full zone configurations via the full_config_sql column in the crdb_internal.zones table, and the new zone_id and subzone_id columns in the crdb_internal.partitions table point to partitions' corresponding zone configurations in crdb_internal.zones

I think the only thing left is matching up `crdb_internal.partitions` with `crdb_internal.zones`, but it looks like that work may happen via #4978 ?

tl;dr: can we close this?  or do we need it to stay open to track the `zones <-> partitions` connection work?

**Comment by ericharmeling** [comment]
@rmloveland I think we can close this.

`crdb_internal.zones` is documented in #5679, and we direct users to query `crdb_internal.partitions` for partition information for "programmatic" purposes in #5722. 

And yes, we plan to go into much more detail about all `crdb_internal` tables with #4978.
